### PR TITLE
Hook up edit contacts on the edit show screen

### PR DESCRIPTION
**Pull request by smn**
Currently clicking on edit contacts leads to a contact selection screen somewhere under the wizard (it should probably be part of the conversation views) and the save button on the screen it redirects to isn't hooked up.

### CODE REVIEW

**Comment by smn** [comment]
@hodgestar is this what you're referring to?

vumi_go

**Comment by hodgestar** [comment]
Yes, but also note the URL contains `/wizard/`. I think this should also be a conversation view.  Maybe 'edit_groups'?

**Comment by smn** [comment]
Uploading of contacts isn't hooked up yet either. Pulling that in with this PR.

**Comment by smn** [comment]
Actually, no, the view logic is a bit complex and potentially allows for creation of new groups. I'm removing that from the template now and leaving contact importing to the contacts view and will create a separate PR.

File: go/contacts/templates/contacts/group_detail.html [hunk 1]
```diff
@@ -33,6 +33,7 @@
         el: '.table-form-view',
         actions: '.table-form-view-buttons button'
     });
+    $('#recMatchFrm').modal('show');
```
Comment (smn) [inline go/contacts/templates/contacts/group_detail.html:36]: This fixed #578.

**Comment by smn** [comment]
Ready for review

File: go/conversation/templates/conversation/edit_groups.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <div class="pull-right">
+        <button class="btn btn-primary" type="submit" value="Save">Save</button>
+    </div>
+{% endblock %}
+
+{% block content_main %}
+<div class="row-fluid">
+    <div class="span12">
+        <div class="indent">
+            <h3>Select contacts</h3>
```
Comment (hodgestar) [inline go/conversation/templates/conversation/edit_groups.html]: Maybe this should read "Select contact groups"?

File: go/conversation/templates/conversation/edit_groups.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+{% block modals %}
+    {{ block.super }}
+    {% include "contacts/includes/tools.html" %}
+{% endblock %}
+
+
+{% block ondomready %}
+    {{ block.super }}
+    // highlight the breadcrumbs.
+    $('.breadcrumbs a').eq(2).addClass('active');
```
Comment (hodgestar) [inline go/conversation/templates/conversation/edit_groups.html]: Does this view actually have breadcrumbs to highlight?

**Comment by hodgestar** [comment]
:+1: other than the question about breadcrumbs and about the wording of "Select contacts".

**Pull request merged**